packages/react-native-windows-init/src/Cli.ts

```diff
@@ -368,12 +408,41 @@ function installReactNativeWindows(
   );
 }
 
+function isMSFTInternal(): boolean {
+  return process.env.USERDNSDOMAIN !== undefined && process.env.USERDNSDOMAIN.endsWith('microsoft.com');
+}
+
+function getRNWInitVersion(): string {
+  try {
+    const pkgJson = require('../package.json');
+    if (pkgJson.name === 'react-native-windows-init' && pkgJson.version !== undefined) {
+      return pkgJson.version;
+    }
+  } catch { }
```

Comment:
Same story here, where we shouldn't catch all. Since this is package local, I would assume it would always work though.

Reply:
As a principle I want to ensure we don't ever crash because we couldn't capture some piece of data that's for telemetry only.